docs/user_guide/alerts_dashboards.md

```diff
@@ -224,62 +159,200 @@ There are several security related reports to choose from and they are all locat
     * Azure Active Directory - ServicePrincipal Change/Update
     * Azure Active Directory - User Change/Update
     * Azure - Current Security Score
-    * Azure - Current Security Score(In Percentage)
+    * Azure - Current Security Score(In Percentage) 
     * External Users Added to Teams
-    * O365: Activity Over World (Unique Users)  
-    * O365: Security & Compliance Center: Alert Details 
-    * Office 365: Service Status
+    * Login by Location
+    * O365: Activity Over World (Unique Users)
+    * O365: Security & Compliance Center: Alert Details
+    * Office 365: Service Status 
     * Office 365/Azure - Failed Logins
-    * Office 365/Azure - Login from Unknown UserId
+    * Office 365/Azure - Login from Unknown UserId 
     * Office 365/Azure - Successful Logins
 
-* Palo Alto Firewall Reports
+## DNS Tracker
+* Alerts:
+    * N/A
+* Dashboards:
+    * DNS Log Volume Over Time
+    * DNS Rquesters 
+    * DNS Server
+    * DNS Server No. of Queries Received 
+    * DNS Server No. of Queries Send to External DNS Server 
+    * Internal DNS Server to DNS Server Requests
+    * Record Types 
+    * Record Types over Time 
+    * Top Categories
+    * Top External DNS Servers being Queried 
+    * Top Non-success Code Queries 
+    * Top Non-success Code Requesters 
+    * Top Queries
+    * Top Requesters 
+
+## Email
+* Microsoft Office 365 Alerts:
+    * Email - Daily Spam Email 
+    * Email - Hourly Increase in Emails Over Baseline 
+* Microsoft Office 365 Dashboards:
+    * N/A
```

Comment:
May be we'll remove this "dashboards" instead of keeing dashboard and mantioning "N/A"